Jenkinsfile

```diff
@@ -850,8 +850,9 @@ try {
                 xcopy C:\\mxnet\\model model /E /I /Y
                 call activate py2
                 pip install mock
-                set PYTHONPATH=${env.WORKSPACE}\\pkg_vc14_cpu\\python
-                del /S /Q ${env.WORKSPACE}\\pkg_vc14_cpu\\python\\*.pyc
+                set PYTHONPATH=${env.WORKSPACE}\\python
+                for /r %%i in (*.dll) do xcopy /Y /C "%%i" .\\python\\mxnet\\
```

Comment:
Would this maybe be a better fit for our makefile?

Reply:
what do you suggest?

Reply:
I also don't get this change. We have MXNET_LIBRARY_PATH to locate the dll.